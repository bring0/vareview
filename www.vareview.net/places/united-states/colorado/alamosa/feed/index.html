<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Alamosa, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/alamosa/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Alamosa Outpatient Clinic</title>
		<link>http://www.vareview.net/places/united-states/colorado/alamosa/outpatient-clinic/alamosa-outpatient-clinic/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/alamosa/outpatient-clinic/alamosa-outpatient-clinic/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 14:52:24 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=907</guid>
		<description><![CDATA[Features Primary care services for veterans in the Alamosa/San Luis Valley area. Handicapped Accessible: full and easy access for all veterans. Personal Care: Highly qualified primary care providers, specialty care referrals to the Colorado Springs CBOC and Denver VA Medical Center. Laboratory: Blood drawing services available. Prescriptions: Routine prescriptions processed through the <a href="http://www.vareview.net/places/united-states/colorado/alamosa/outpatient-clinic/alamosa-outpatient-clinic/">read more</a>]]></description>
				<content:encoded><![CDATA[<h3>Features</h3>
<ul>
<li>Primary care services for veterans in the Alamosa/San Luis Valley area.</li>
<li>Handicapped Accessible: full and easy access for all veterans.</li>
<li>Personal Care: Highly qualified primary care providers, specialty care referrals to the Colorado Springs CBOC and Denver VA Medical Center.</li>
<li>Laboratory: Blood drawing services available.</li>
<li>Prescriptions: Routine prescriptions processed through the mail or <a href="http://www.myhealth.va.gov/">My Health<em>e</em>Vet</a>.</li>
</ul>
<h3><a id="driving" class="bookmark" title="driving" name="driving"></a>Driving Directions</h3>
<p>From US-160 turn RIGHT onto MAIN ST/US-160. Continue to follow US-160 turn RIGHT onto DEL SOL DR</p>
<h3><a id="parking" class="bookmark" title="parking" name="parking"></a>Parking</h3>
<ul>
<li>Free on premises</li>
</ul>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/alamosa/outpatient-clinic/alamosa-outpatient-clinic/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
