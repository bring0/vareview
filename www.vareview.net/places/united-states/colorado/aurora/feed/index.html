<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Aurora, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/aurora/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Aurora Outpatient Clinic</title>
		<link>http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/aurora-outpatient-clinic/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/aurora-outpatient-clinic/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 14:53:39 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=908</guid>
		<description><![CDATA[Features Primary care services for veterans in Denver metro area. Handicapped Accessible: full and easy access for all veterans. Personal Care: Highly qualified primary care providers, specialty care referrals to the Denver VA Medical Center. Laboratory: Blood drawing services available. Prescriptions: Routine prescriptions processed through the mail or My HealtheVet. <a href="http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/aurora-outpatient-clinic/">read more</a>]]></description>
				<content:encoded><![CDATA[<h3>Features</h3>
<ul>
<li>Primary care services for veterans in Denver metro area.</li>
<li>Handicapped Accessible: full and easy access for all veterans.</li>
<li>Personal Care: Highly qualified primary care providers, specialty care referrals to the Denver VA Medical Center.</li>
<li>Laboratory: Blood drawing services available.</li>
<li>Prescriptions: Routine prescriptions processed through the mail or <a href="http://www.myhealth.va.gov/">My Health<em>e</em>Vet</a>.</li>
</ul>
<p><a id="driving" class="bookmark" title="driving" name="driving"></a></p>
<h3><a id="parking" class="bookmark" title="parking" name="parking"></a>Parking</h3>
<ul>
<li>Free on premises</li>
</ul>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/aurora-outpatient-clinic/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
		<item>
		<title>Jewell Clinic</title>
		<link>http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/jewell-clinic/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/jewell-clinic/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 14:12:13 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=902</guid>
		<description><![CDATA[This clinic is home to the Physical Medicine and Rehabilitation Service, the Prosthetic Treatment Center, the Health Information Call Center and the Rural Telehealth Center. Services Polytrauma Program Assistive Technology Program Drivers&#8217; Training Rehab Outpatient Recreation Therapy Regional Amputation Center (RAC) Participation Enhancement Speech Pathology and Voice Lab Audiology Wheelchair <a href="http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/jewell-clinic/">read more</a>]]></description>
				<content:encoded><![CDATA[<p>This clinic is home to the Physical Medicine and Rehabilitation Service, the Prosthetic Treatment Center, the Health Information Call Center and the Rural Telehealth Center.</p>
<h3>Services</h3>
<ul>
<li>Polytrauma Program</li>
<li>Assistive Technology Program</li>
<li>Drivers&#8217; Training Rehab</li>
<li>Outpatient Recreation Therapy</li>
<li>Regional Amputation Center (RAC)</li>
<li>Participation Enhancement</li>
<li>Speech Pathology and Voice Lab</li>
<li>Audiology</li>
<li>Wheelchair Program</li>
<li>Orthotic and Prosthetic Lab</li>
<li>Shoe Clinic</li>
<li>Brace Clinic</li>
<li>Amputee Clinic</li>
<li>Gait Lab</li>
<li>Telemedicine Wheelchair, Amputee  and Brace Clinics</li>
<li>Visual Impairment Services Outpatient Rehabilitation Program (VISOR)</li>
<li>MOVE! Program</li>
<li>Telehealth Section</li>
</ul>
<h3>Driving Directions</h3>
<ul>
<li>From the North: Exit I-225 South at Mississippi.  Turn left (East) on Mississippi. Turn Right (South) on Sable Blvd.  Turn Right (West) on Jewell Ave. The clinic is on the LEFT.</li>
<li>From the South: Exit I-225 North at Iliff.  Turn Right (East) on Iliff.  Turn Left (North) on Sable Blvd.  Turn Left (West) on Jewell Ave. The clinic is on the LEFT.</li>
</ul>
<h3>Parking</h3>
<ul>
<li>Free on premises</li>
</ul>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/aurora/outpatient-clinic/jewell-clinic/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
