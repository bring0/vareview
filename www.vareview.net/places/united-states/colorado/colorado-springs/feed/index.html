<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Colorado Springs, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/colorado-springs/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>PFC Floyd K. Lindstrom Outpatient Clinic at Colorado Springs</title>
		<link>http://www.vareview.net/places/united-states/colorado/colorado-springs/outpatient-clinic/pfc-floyd-k-lindstrom-outpatient-clinic-at-colorado-springs/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/colorado-springs/outpatient-clinic/pfc-floyd-k-lindstrom-outpatient-clinic-at-colorado-springs/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 14:55:12 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=909</guid>
		<description><![CDATA[Features * Primary care services for veterans in the Colorado Springs area. * Handicapped Accessible: full and easy access for all veterans. * Personal Care: Highly qualified primary and specialty care and mental health providers with referrals to the Denver VA Medical Center. * Laboratory: Blood drawing services available. * <a href="http://www.vareview.net/places/united-states/colorado/colorado-springs/outpatient-clinic/pfc-floyd-k-lindstrom-outpatient-clinic-at-colorado-springs/">read more</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Features</strong></p>
<p>* Primary care services for veterans in the Colorado Springs area.<br />
* Handicapped Accessible: full and easy access for all veterans.<br />
* Personal Care: Highly qualified primary and specialty care and mental health providers with referrals to the Denver VA Medical Center.<br />
* Laboratory: Blood drawing services available.<br />
* Prescriptions: Routine prescriptions processed through the mail or <a href="https://www.myhealth.va.gov/index.html">My HealtheVet.</a></p>
<p><strong>Services at the New Colorado Springs Clinic:</strong></p>
<p>*Optometry, Audiology, Pharmacy, Occupational Therapy, Physical Therapy, Laboratory, Compensation and Pension, CWT, Homeless, and the Pre and Post-Surgical Clinic will be located on the first floor.</p>
<p>*Primary Care services, OEF/OIF, wound care, and MOVE classroom will be located on the second floor. Dental, Mental Health, and Substance Abuse Clinics will be located on the third floor.</p>
<p>*New Services: MRI, CT, ultrasound, and x-ray services as well as the Veterans Benefits Administration (VBA) will be located on the first floor.</p>
<p>*The new clinic will be a tobacco-free campus. Please help us make this move to an improved health environment by refraining from using any form of tobacco on the campus, including the parking lot.</p>
<p><strong>Driving Directions</strong></p>
<p>* Take I-25 towards Colorado Springs, and exit 145 for CO-38 E/Fillmore Street. Turn right onto W Fillmore St, and head West for approx. Â½ mile. Make a left turn at the light at Centennial Boulevard (by King Soopers Plaza) and the clinic is on your left.</p>
<p><strong>Parking<br />
</strong><br />
* Parking is available in the front of the building to include Disabled Parking Spaces.</p>
<p><strong>Hours</strong><br />
* Monday &amp; Tuesday 7:30am &#8211; 4:30pm<br />
* Wednesday &#8211; Saturday 7:30am &#8211; 6:30pm</p>
<p>Colorado Springs Transit will provide a bus stop on Centennial Blvd. For more information please view details at <a href="https://www.springsgov.com">https://www.springsgov.com</a> or call (719) 385-7433.</p>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/colorado-springs/outpatient-clinic/pfc-floyd-k-lindstrom-outpatient-clinic-at-colorado-springs/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
