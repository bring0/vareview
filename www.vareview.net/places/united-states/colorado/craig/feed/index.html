<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Craig, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/craig/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Craig, Colorado</title>
		<link>http://www.vareview.net/places/united-states/colorado/craig/outpatient-clinic/craig-colorado/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/craig/outpatient-clinic/craig-colorado/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 14:57:26 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=910</guid>
		<description><![CDATA[Features Primary care services for veterans in northwestern Colorado and southwestern Wyoming. Services offered: Primary Care Visit &#38; Follow-up Health Screenings Mental Health Visits Wound Care Various Pre-Op Screenings Post Op Management Suture Removal Nurse Clinic Visits EKG Immunizations Patient Education Medication Management Blood Pressure Management Diabetes Management Handicapped Accessible: full <a href="http://www.vareview.net/places/united-states/colorado/craig/outpatient-clinic/craig-colorado/">read more</a>]]></description>
				<content:encoded><![CDATA[<h3>Features</h3>
<ul>
<li>Primary care services for veterans in northwestern Colorado and southwestern Wyoming.</li>
<li>Services offered:
<ul>
<li>Primary Care Visit &amp; Follow-up</li>
<li>Health Screenings</li>
<li>Mental Health Visits</li>
<li>Wound Care</li>
<li>Various Pre-Op Screenings</li>
<li>Post Op Management</li>
<li>Suture Removal</li>
<li>Nurse Clinic Visits</li>
<li>EKG</li>
<li>Immunizations</li>
<li>Patient Education</li>
<li>Medication Management</li>
<li>Blood Pressure Management</li>
<li>Diabetes Management</li>
</ul>
</li>
<li>Handicapped Accessible: full and easy access for all veterans.</li>
<li>Laboratory: Blood drawing services available.</li>
<li>Prescriptions: Routine prescriptions processed through the mail or <a href="http://www.myhealth.va.gov/">My Health<em>e</em>Vet</a>.</li>
</ul>
<h3>Appointments</h3>
<p>To make an appointment call (970) 824-6721.</p>
<h3>Driving Directions</h3>
<p><a id="parking" class="bookmark" title="parking" name="parking"></a>When travelling West on Victory Way, turn right onto Russell Street and proceed past 7th Street.  We are located on the right just before 8th Street, on the corner of 8th St. and Russell St., Suite 400.</p>
<p>When travelling East on Victory Way, turn left onto Russell Street and proceed past 7th Street.  We are located on the right just before 8th Street, on the corner of 8th St. and Russell St., Suite 400.</p>
<h3>Parking</h3>
<ul>
<li>Free on premises</li>
</ul>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/craig/outpatient-clinic/craig-colorado/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
