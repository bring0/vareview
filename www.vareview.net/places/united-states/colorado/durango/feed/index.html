<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Durango, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/durango/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Durango Clinic</title>
		<link>http://www.vareview.net/places/united-states/colorado/durango/outpatient-clinic/durango-clinic/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/durango/outpatient-clinic/durango-clinic/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 14:59:51 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=911</guid>
		<description><![CDATA[The Durango Contract Community Based Outpatient Clinic (CBOC) is designed to deliver primary care services to eligible Veterans. Services and Scope of Care: Blood Draws (By appt. only) Routine Lab Tests EKGs (Electrocardiograms) Social Work Services Evaluations by Doctors Location: 1970 East Third Avenue, Suite 102, Durango, CO  81301 Eligibility:  <a href="http://www.vareview.net/places/united-states/colorado/durango/outpatient-clinic/durango-clinic/">read more</a>]]></description>
				<content:encoded><![CDATA[<p>The Durango Contract Community Based Outpatient Clinic (CBOC) is designed to deliver primary care services to eligible Veterans.</p>
<p><strong>Services and Scope of Care:</strong></p>
<ul>
<li>Blood Draws (By appt. only)</li>
<li>Routine Lab Tests</li>
<li>EKGs (Electrocardiograms)</li>
<li>Social Work Services</li>
<li>Evaluations by Doctors</li>
</ul>
<p><strong>Location:</strong> 1970 East Third Avenue, Suite 102, Durango, CO  81301</p>
<p><strong>Eligibility:</strong>  For information about enrollment in the Durango CBOC call Eligibility and Enrollment at (505) 265-1711, ext. 2741.</p>
<ul>
<li>Application for Care can be faxed to:  (505) 256-2808</li>
<li>Co-Pay:  Co-Pay may apply.</li>
</ul>
<p><strong>Other Pertinent Information:</strong></p>
<ul>
<li>For appointments, call the Clinic at (970) 247-2214.</li>
<li>To assist in your care and treatment, please bring all of your medications to your first appointment.</li>
</ul>
<p><strong>Emergency Care:</strong>  No emergency care is available at this facility.  For after-hours assistance, call 1-866-862-7863 or for emergency care call 911.</p>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/durango/outpatient-clinic/durango-clinic/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
