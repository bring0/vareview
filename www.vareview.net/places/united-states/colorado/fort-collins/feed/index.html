<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Fort Collins, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/fort-collins/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Fort Collins VA MSOC</title>
		<link>http://www.vareview.net/places/united-states/colorado/fort-collins/outpatient-clinic/fort-collins-va-msoc/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/fort-collins/outpatient-clinic/fort-collins-va-msoc/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 15:01:10 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=912</guid>
		<description><![CDATA[Fort Collins, Colorado MSOC The Fort Collins VA Clinic, an outpatient clinic managed from the Cheyenne VA Medical Center, at nearly 10,000 square feet, fosters the ability for veterans in the northern portions of Colorado to continue to receive high quality VA clinical services. Services provided at the Fort Collins <a href="http://www.vareview.net/places/united-states/colorado/fort-collins/outpatient-clinic/fort-collins-va-msoc/">read more</a>]]></description>
				<content:encoded><![CDATA[<h3>Fort Collins, Colorado MSOC</h3>
<p>The Fort Collins VA Clinic, an outpatient clinic managed from the Cheyenne VA Medical Center, at nearly 10,000 square feet, fosters the ability for veterans in the northern portions of Colorado to continue to receive high quality VA clinical services.</p>
<p>Services provided at the Fort Collins VA Clinic include: primary care, mental health counseling, audiology, nutrition counseling as well as physical therapy and occupational therapy services once per month. Additionally, the on-site laboratory provides routine blood draws.Â  The Vet Center will continue to provide regular individual and group counseling services at the new site.</p>
<p>The new location of the Fort Collins VA Clinic is at 2509 Research Blvd., just north of West Drake Road.</p>
<h3>
Multi-Speciality Outpatient Clinic Features:</h3>
<p>Primary care services for veterans in the Fort Collins area.<br />
Behavioral Health services including individual, group and family counseling.<br />
Handicapped Accessible: full and easy access for all veterans.<br />
Personal Care: Highly qualified primary care providers, specialty care referrals to the Cheyenne VA Medical Center.<br />
Laboratory: Blood drawing services available.<br />
Prescriptions: Routine prescriptions processed through the mail or My HealtheVet.</p>
<h3>
Driving Directions</h3>
<p>From I-25, exit on Prospect Avenue west. Continue until College Avenue, turn south onto College Ave. until West Drake Road. Turn west onto West Drake Rd., then north onto Research Blvd.</p>
<p>Continue north on Research Blvd. about 100 yards, the VA Clinic is on the west side of that point.</p>
<h3>Parking</h3>
<p>Free on premises</p>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/fort-collins/outpatient-clinic/fort-collins-va-msoc/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
