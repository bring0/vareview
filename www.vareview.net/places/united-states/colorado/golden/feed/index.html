<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Places in Golden, Colorado, United States | VA Review</title>
	<atom:link href="http://www.vareview.net/places/united-states/colorado/golden/feed/" rel="self" type="application/rss+xml" />
	<link>http://www.vareview.net</link>
	<description></description>
	<lastBuildDate>Tue, 02 Aug 2016 15:02:43 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	
	<item>
		<title>Golden Outpatient Clinic</title>
		<link>http://www.vareview.net/places/united-states/colorado/golden/outpatient-clinic/golden-outpatient-clinic/</link>
		<comments>http://www.vareview.net/places/united-states/colorado/golden/outpatient-clinic/golden-outpatient-clinic/#reviews</comments>
		<pubDate>Fri, 22 Jul 2016 15:03:39 +0000</pubDate>
		<dc:creator><![CDATA[Jason W.]]></dc:creator>
		
		<guid isPermaLink="false">http://www.vareview.net/?post_type=gd_place&#038;p=913</guid>
		<description><![CDATA[Features Primary care services for veterans in the Denver Metro, Front Range and Mountain communities. Handicapped Accessible: full and easy access for all veterans with parking spots located near the front entrance to the clinic Personal Care: Highly qualified primary care providers, specialty care referrals to the Denver VA Medical <a href="http://www.vareview.net/places/united-states/colorado/golden/outpatient-clinic/golden-outpatient-clinic/">read more</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Features</strong></p>
<ul>
<li>Primary care services for veterans in the Denver Metro, Front Range and Mountain communities.</li>
<li>Handicapped Accessible: full and easy access for all veterans with parking spots located near the front entrance to the clinic</li>
<li>Personal Care: Highly qualified primary care providers, specialty care referrals to the Denver VA Medical Center.</li>
<li>Laboratory &amp; Radiology services are available at this clinic.</li>
<li>Physical Therapy: A wide range of Physical Therapy services will be offered at the clinic. You <em>must</em> have a Physical Therapy consult from your VA provider to receive therapy.</li>
<li>Other Services available include Nutritional Medicine, Audiology, and Travel Vouchers.</li>
<li>Prescriptions: Routine prescriptions processed through the mail, in the community via Heritage Pharmacy, or <a href="https://www.myhealth.va.gov/index.html">My HealtheVet.</a></li>
<li>Compensation &amp; Pension: This clinic is designed to support this program by having audiology, laboratory and radiological services necessary for these evaluations.</li>
</ul>
<p>The new clinic is a tobacco-free campus which includes e-cigarettes. Please help us make this move to an improved health environment by not using tobacco in any form on the campus, including the parking lot.</p>
<p><strong>Driving Directions</strong></p>
<p>From US-6 W / W 6<sup>th</sup> Ave turn right onto Johnson Road. Follow Johnson Road north to 10<sup>th</sup> Avenue. The Golden VA Clinic will be on the right-hand sign, in the center building of the office complex.</p>
<p><strong>Parking</strong></p>
<ul>
<li>Ample handicap and regular patient parking is provided both in front of the clinic and to the east.</li>
</ul>
<p><strong>Hours</strong></p>
<ul>
<li>Primary Care and Mental Health clinic (Second floor) hours of operation &#8211; Monday through Friday 7:30 am to 4:30 pm.</li>
<li>Compensation and Pension Medical Exam section (First floor) will be open Monday through Friday from 7:00 AM to 7:00 PM, and Saturdays from 7:30 am to 4:30 pm.</li>
<li>Laboratory, Radiology, Physical Therapy, and other ancillary departments will be open from 8:00 AM to 3:30 PM. The clinic will be closed all Federal Holidays.</li>
<li>Please call the clinic at 303-914-2680 for appointments, and the Nurse Information Call Center at 303-399-8020 then option 2 for questions about your medical care. After hours this number will be answered by triage nurses who will review your reported concern/issue and make appropriate referral for healthcare.</li>
</ul>
]]></content:encoded>
			<wfw:commentRss>http://www.vareview.net/places/united-states/colorado/golden/outpatient-clinic/golden-outpatient-clinic/feed/</wfw:commentRss>
		<slash:comments></slash:comments>
		</item>
	</channel>
</rss>
